### ISSUE TITLE
javascript version of .toggle();

### ISSUE THREAD

**Issue by david-hc** [HubSpot/youmightnotneedjquery#84]
I new to github and dont know almost everything. I cannot find .toggle() in youmighnotneedjquery.com. I have similar js to do that. I fight jQuery

```
el.addEventListener('click', function() {
  if(el2.offsetWidth > 0 || el2.offsetHeight > 0) {
    el2.style.display = 'none';
  } else {
    el2.style.display = '';
  }
});
```

**Comment by dmitry** [comment]
@david-hc how is this related to `toggle`?

**Comment by Abbondanzo** [comment]
Closed by #253
